cachito/workers/pkg_managers/general_js.py

```diff
@@ -55,9 +55,17 @@ def download_dependencies(request_id, deps):
     conf = get_worker_config()
     with tempfile.TemporaryDirectory(prefix="cachito-") as temp_dir:
         npm_rc_file = os.path.join(temp_dir, ".npmrc")
+        if conf.cachito_nexus_ca_cert and os.path.exists(conf.cachito_nexus_ca_cert):
```

Comment:
If it's set, but the path doesn't exist, should we at least log a warning? Maybe even raise an exception?

Reply:
It has a default value so it's not unusual if it's set but the path doesn't exist:
https://github.com/release-engineering/cachito/blob/cb4d01ec113f3e48dcbc8ab4efeca22744c24cc0/cachito/workers/config.py#L26

I was just being more defensive by making sure it wasn't unset by the user before checking if the path exists.